apps/examples/smartfs/Makefile

```diff
@@ -16,7 +16,7 @@
 #
 ###########################################################################
 
-MENUDESC = "SmartFs Test Applications"
+MENUDESC = "jSmartFs Test Applications"
```

Comment:
The subfolder name should be changed from smartfs to jsmartfs
Here the spelling will be "jsmartfs" only